tools/docker-dev/Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM docker:latest as docker
+FROM docker:stable as docker
 
-FROM ubuntu:latest
+FROM ubuntu:xenial
 ARG goversion=1.12.5
 ARG user
 ARG group
 ARG uid=1000
 ARG gid=1000
 
 # Install development packages.
-RUN apt-get update && apt-get -qqy upgrade && apt-get -qqy install \
+# hadolint ignore=DL3008
+RUN apt-get update && apt-get -qqy install --no-install-recommends \
 autoconf \
```

Comment:
As a first shot, here are the versions for these packages:
```
autoconf=2.69-9 \
autotools-dev=20150820.1 \
build-essential=12.1ubuntu2 \
curl=7.47.0-1ubuntu2.13 \
git=1:2.7.4-0ubuntu1.6 \
libtool=2.4.6-0.1 \
lsb-release=9.20160110ubuntu0.2 \
make=4.1-6 \
sudo=1.8.16-0ubuntu1.7 \
bash-completion=1:2.1-4.2ubuntu1.1 \
jq=1.5+dfsg-1ubuntu0.1 \
tmux=2.1-3build1 \
vim=2:7.4.1689-3ubuntu1.3 \
```

Now, I don't know what to do about the tons of other dependencies that are installed by those packages...

Reply:
Add `ca-certificates` to fix downloading `go` below:

```
autoconf=2.69-9 \
autotools-dev=20150820.1 \
build-essential=12.1ubuntu2 \
ca-certificates=20170717~16.04.2 \
curl=7.47.0-1ubuntu2.13 \
git=1:2.7.4-0ubuntu1.6 \
libtool=2.4.6-0.1 \
lsb-release=9.20160110ubuntu0.2 \
make=4.1-6 \
sudo=1.8.16-0ubuntu1.7 \
bash-completion=1:2.1-4.2ubuntu1.1 \
jq=1.5+dfsg-1ubuntu0.1 \
tmux=2.1-3build1 \
vim=2:7.4.1689-3ubuntu1.3 \
```

Reply:
Then you should be able to remove the `# hadolint ignore=DL3008` above.